bridgev2/portal.go

```diff
@@ -272,12 +272,16 @@ func (br *Bridge) GetExistingPortalByKey(ctx context.Context, key networkid.Port
 }
 
 func (portal *Portal) queueEvent(ctx context.Context, evt portalEvent) {
-	select {
-	case portal.events <- evt:
-	default:
-		zerolog.Ctx(ctx).Error().
-			Str("portal_id", string(portal.ID)).
-			Msg("Portal event channel is full")
+	if PortalEventBuffer == 0 {
```

Comment:
PortalEventBuffer probably needs to be turned into a `var` rather than `const`

Reply:
Oh good spot, fixed!